.github/workflows/keyvalue-provider-release.yml

```diff
@@ -0,0 +1,88 @@
+name: INMEMORY-KEYVALUE-RELEASE
+
+on:
+  push:
+    tags:
+    - 'inmemory-keyvalue-v*'
+
+env:
+  CARGO_TERM_COLOR: always
+  working-directory: ./inmemory-keyvalue
+  DESTINATION: inmemory-keyvalue.par.gz #aka the filename of the parJEEzy
+  WASH_ISSUER_KEY: ${{ secrets.WASMCLOUD_ACCOUNT_OFFICIAL }}
+  WASH_SUBJECT_KEY: ${{ secrets.WASMCLOUD_INMEMORY-KEYVALUE }}
```

Comment:
gh actions doesn't support `-` in the secret names. Please update `WASMCLOUD_INMEMORY-KEYVALUE` to `WASMCLOUD_INMEMORY_KEYVALUE

Reply:
Done! Thanks 😄